src/Panel/PanelHeader.d.ts

```diff
@@ -0,0 +1,18 @@
+import { PureComponent, ReactChildren, ReactNode } from 'react';
+import { PanelProps } from './Panel';
+
+
+// TODO: generalize with Panel.Props
+export type PanelHeaderProps = {
+  collapse?: {
+    direction: PanelProps.CollapseDirection,
+    onToggleExpanded: () => void,
+    isExpanded?: boolean
+  },
+  size?: PanelProps.HeaderSize,
+  content?: any,
+  title?: ReactNode | string, // TODO(typo): wtf
+  menu?: ReactChildren
```

Comment:
and this one `ReactChildren`?
I think there is a lot of confusion on how to translate `t.ReactChildren` in Typescript.

@giogonzo do we have a guideline on how to handle these cases?